CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## [1.0.120230112]
```

Comment:
Can we update this version to match package.json?

Reply:
Actually package json has to be updated if it is not same version as change log. It's the date we release Vs code.

Reply:
Thanks for catching this and if you don't mind can you update them as well ?